packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormUpdateRecord.tsx

```diff
@@ -31,9 +36,23 @@ type WorkflowEditActionFormUpdateRecordProps = {
 type UpdateRecordFormData = {
   objectName: string;
   objectRecordId: string;
+  fieldsToUpdate: string[];
   [field: string]: unknown;
 };
 
+const AVAILABLE_FIELD_METADATA_TYPES = [
+  FieldMetadataType.Text,
+  FieldMetadataType.Number,
+  FieldMetadataType.Date,
+  FieldMetadataType.Boolean,
+  FieldMetadataType.Select,
+  FieldMetadataType.MultiSelect,
+  FieldMetadataType.Emails,
+  FieldMetadataType.Links,
+  FieldMetadataType.FullName,
+  FieldMetadataType.Address,
+];
+
```

Comment:
What is the reason behind this static list of field types?

Reply:
@Devessier I do not want the user to select a field a do not see the form to be updated. That would look weird. Some fields such as Rating may not be implemented before we get our first users in Beta. I prefer adding this list and remove it once we have all fields

Reply:
Makes sense.